intel_extension_for_transformers/neural_chat/examples/serving/triton_inference_sever/hpu/README.md

```diff
@@ -0,0 +1,96 @@
+# Serving NeuralChat Text Generation with Triton Inference Server on HPU
+
+Nvidia Triton Inference Server is a widely adopted inference serving software. We also support serving and deploying NeuralChat models with Triton Inference Server.
+
+## Prepare serving scripts
+
+```bash
+cd <path to intel_extension_for_transformers>/neural_chat/examples/serving/triton_inference_server
+mkdir -p models/text_generation/1/
+cp ../../../serving/triton/text_generation/model.py models/text_generation/1/model.py
+cp ../../../serving/triton/text_generation/client.py models/text_generation/1/client.py
+cp ../../../serving/triton/text_generation/config.pbtxt models/text_generation/config.pbtxt
+```
+
+Make sure `KIND_CPU` is used for instance_group in `config.pbtxt`.
```

Comment:
The `config.pbtxt` for HPU is different with default file which is for CPU.

Reply:
Should add the 'count' parameter for multi-cards use case.

Reply:
ok, will add another config file for multi-cards.